src/main/java/org/jabref/gui/cleanup/CleanupDialog.java

```diff
@@ -9,14 +10,20 @@
 import org.jabref.preferences.FilePreferences;
 
 public class CleanupDialog extends BaseDialog<CleanupPreferences> {
-
     public CleanupDialog(BibDatabaseContext databaseContext, CleanupPreferences initialPreset, FilePreferences filePreferences) {
         setTitle(Localization.lang("Cleanup entries"));
-        getDialogPane().setPrefSize(600, 600);
+        getDialogPane().setPrefSize(600, 650);
```

Comment:
I know that we actually have one user with an old screen with a rather small window size. We should defenitly keep it in mind that some user - maybe also with obscure window sizes on screens in portrait mode - should still be able to use jabref.

Reply:
Size is ok on my small laptop with 1366x768 resolution. Barely, within the margins, but it is ok. See screenshot in other comment.

Reply:
@calixtus I am using Ubuntu with a screen resolution of 1920x1080 pixels (508x285 millimeters), and when I launched the *CleanupDialog* with the the **preferred height** of **600**, I noticed that the content of the dialog didn't show up entirely, and as a result the scroll bar appeared. **(as shown in the screenshot below &#8595;)**

![Screenshot from 2022-10-13 21-32-00](https://user-images.githubusercontent.com/82417779/195709234-3c099d6f-dd1e-4afb-b9ed-fa94bb31d7ec.png)

This is why I increased the **pref height** to **650** so the content shows up entirely in the first launch, and the scroll bar only shows up when needed. (when the window is resized) &#8595;

![Screenshot from 2022-10-13 21-30-20](https://user-images.githubusercontent.com/82417779/195710501-8e33ed95-4978-4726-9a1d-2647bc9f053e.png)

I might be wrong in my choice when considering different window sizes.